aten/src/ATen/core/function_schema_inl.h

```diff
@@ -74,6 +105,80 @@ inline std::string FunctionSchema::formatTypeMismatchMsg(
       *this);
 }
 
+inline bool FunctionSchema::isBackwardCompatibleWith(
+    const FunctionSchema& old,
+    std::ostream* why_not) const {
+  if (!(name() == old.name()
+        && overload_name() == old.overload_name()
+        && is_vararg() == old.is_vararg()
```

Comment:
I'd add a comment here that technically adding vararg is backward compat, but it's internal only, so it'd fine to ignore it for now

Reply:
added